README.md

```diff
@@ -10,36 +10,18 @@ Can be used with any domain, or CSV where domains are the first column, such as
 
 The requirements here can be quite diverse, because this tool is just a coordinator for other tools. Communication between tools is handled via CLI and STDOUT.
 
-The overall tool requires **Python 3**. To install dependencies:
+The overall tool requires **Python 3** and **Docker**. To install dependencies:
 
 ```bash
 pip install -r requirements.txt
 ```
```

Comment:
@adelevie Since the tool now requires Docker for the individual scanners, what do you think about simplifying the instructions to run the `./scan` through Docker exclusively?